### DISCUSSION TITLE
Output error

### DISCUSSION THREAD

**Discussion by ayoubackup** [projectdiscovery/nuclei#3495] [General]
Hi projectdiscovery

i have run nuclei for big list and sometimes i got output :

2023/04/02 04:25:02 Unsolicited response received on idle HTTP channel starting with "HTTP/1.1 200 BANNED\r\n"; err=<nil>

i try to filter it using grep but i got same error

cat list.txt | httpx.exe -t 200 -silent | nuclei -t mytemplate.yaml -silent -timeout 1 | grep -v "Unsolicited response"

is there any solustion to stop this noise output

NOTE: i doubt because i ' m not using --no-color flag !

**Comment by olearycrew** [comment]
There is an open issue for this: https://github.com/projectdiscovery/nuclei/issues/3462.

You can follow that issue for updates.
